data/migrations/v0.9.0-cleanup.sql

```diff
@@ -2,3 +2,25 @@
 DROP FUNCTION IF EXISTS mz_calculate_poi_level(
   text, text, text, text, text, text, text, text, text, text, text, text, text,
   text, text, text, text, text, hstore, real);
+
```

Comment:
Should we also drop the 3 landuse-related functions here? `mz_calculate_is_landuse`, `mz_calculate_landuse_kind` & `mz_calculate_landuse_min_zoom`?